src/Sarif.Viewer.VisualStudio.Core/Tags/SarifLocationErrorTagger.cs

```diff
@@ -65,8 +65,6 @@ public IEnumerable<ITagSpan<IErrorTag>> GetTags(NormalizedSnapshotSpanCollection
         {
             if (this.tagsDirty)
             {
-                this.tagsDirty = false;
-
```

Comment:
wondering if any performance impact after deleting this line. `this.tagsDirty` will always be true and calculate `this.currentTags` every time.

Reply:
I just tested and I did not notice any performance degradation/change when comparing builds, however I also did not notice any difference in functionality so I have added the line back.